packages/std/src/math/uint128.rs

```diff
@@ -297,6 +296,15 @@ impl<'a> Sum<&'a Uint128> for Uint128 {
     }
 }
 
+#[allow(clippy::all)]
```

Comment:
I'm not sure I'm happy with this workaround - a little hacky, isn't it? The problem is the code `construct_uint!` expands to has a bunch of stuff clippy lints don't like.

Reply:
Maybe a little comment explaining this would be nice. Otherwise I see not issue here as all U256 business is purely private.